src/server/land-grants/controllers/remove-action-page.controller.js

```diff
@@ -81,11 +90,13 @@ export default class RemoveActionPageController extends QuestionPageController {
    * @returns {object|null} - Validation error or null if valid
    */
   validatePostPayload(payload) {
-    const { removeAction } = payload
+    const { remove } = payload
 
-    if (removeAction === undefined) {
+    if (remove === undefined) {
       return {
-        errorMessage: 'Please select if you want to remove the action'
+        errorMessage: this.actionDescription
+          ? `Select yes to remove [${this.actionDescription}] from land parcel [${this.parcel}]`
+          : `Select yes to remove land parcel [${this.parcel}] from this application`
```

Comment:
I dont think you need square brackets in the error message
```suggestion
          ? `Select yes to remove ${this.actionDescription} from land parcel ${this.parcel}`
          : `Select yes to remove land parcel ${this.parcel} from this application`
```